NoSuchMethodError: com.android.build.gradle.tasks.ProcessAndroidResources.getPackageForR()Ljava/lang/String;

```
Caused by: java.lang.NoSuchMethodError: com.android.build.gradle.tasks.ProcessAndroidResources.getPackageForR()Ljava/lang/String;
        at butterknife.plugin.ButterKnifePlugin$configureR2Generation$1$1.execute(ButterKnifePlugin.kt:59)
        at butterknife.plugin.ButterKnifePlugin$configureR2Generation$1$1.execute(ButterKnifePlugin.kt:18)
```

It seems that AGP 3.1 has removed a method that the plugin uses.

--- a/butterknife-gradle-plugin/src/main/java/butterknife/plugin/ButterKnifePlugin.kt
+++ b/butterknife-gradle-plugin/src/main/java/butterknife/plugin/ButterKnifePlugin.kt
@@ -56,7 +56,7 @@ class ButterKnifePlugin : Plugin<Project> {
         // Though there might be multiple outputs, their R files are all the same. Thus, we only
         // need to configure the task once with the R.java input and action.
         if (once.compareAndSet(false, true)) {
-          val rPackage = processResources.packageForR
+          val rPackage = processResources.originalApplicationId
           val pathToR = rPackage.replace('.', File.separatorChar)
           val rFile = processResources.sourceOutputDir.resolve(pathToR).resolve("R.java")
 

